src/App.tsx

```diff
@@ -1,31 +1,36 @@
 import React, { useCallback, useState } from "react";
 import "@shopify/polaris/styles.css";
 import { Frame, TopBar, Navigation } from "@shopify/polaris";
-import {
-  HomeMajorMonotone,
-} from "@shopify/polaris-icons";
-import Home from "./components/Home";
+import { HomeMajor } from "@shopify/polaris-icons";
```

Comment:
Compiler couldn't find `HomeMajorMonotone` in `@shopify/polaris-icons`, wouldn't compile ...